### PR TITLE
fix cupy.random.dirichlet to behave same as numpy.random.dirichlet

### PR DESCRIPTION
**Pull request by YoshikawaMasashi**
for now
```
>>> import numpy
>>> import cupy
>>> numpy.random.dirichlet([1,1,1], (100,3)).shape
(100, 3, 3)
>>> cupy.random.dirichlet([1,1,1], (100,3)).shape
(100, 3)
```

I fixed this.

### CODE REVIEW

**Comment by YoshikawaMasashi** [comment]
jenkins, test this please.

**Comment by YoshikawaMasashi** [comment]
jenkins, test this please.

File: tests/cupy_tests/random_tests/test_distributions.py [hunk 1]
```diff
@@ -14,6 +14,17 @@
 _int_dtypes = _signed_dtypes + _unsigned_dtypes
 
 
+class RandomDistributionsTestCase(unittest.TestCase):
+    def check_distribution(self, dist_name, params, dtype):
+        cp_params = {k: cupy.asarray(params[k]) for k in params}
+        np_out = numpy.random.__dict__[dist_name](
```
Comment (okuta) [inline tests/cupy_tests/random_tests/test_distributions.py]: How about using `getattr`?
``` np_out = getattr(numpy.random, dist_name)(```

File: tests/cupy_tests/random_tests/test_distributions.py [hunk 2]
```diff
@@ -14,6 +14,17 @@
...
+            size=self.shape, **params).astype(dtype)
+        cp_out = distributions.__dict__[dist_name](
```
Comment (okuta) [inline tests/cupy_tests/random_tests/test_distributions.py]: ditto

**Comment by YoshikawaMasashi** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
LGTM!

**Pull request merged**